akka-actor-typed/src/main/scala/akka/actor/typed/eventstream/EventStream.scala

```diff
@@ -61,6 +65,14 @@ object EventStream {
   /**
    * Unsubscribe an actor ref from the event stream
    * by sending this command to the [[akka.actor.typed.ActorSystem.eventStream]].
+   * <p>
+   *  <strong>
+   *   Unsubscribing an unsubscribed actor ref will triggers the expensive
+   * check of "whether actor ref still has subscriptions" in the [[akka.event.EventStream]],
+   * even though the operation is not performed on the EventStream itself,
+   * but it affects the timely "watch" of new subscribers by the [[akka.event.EventStream]].
+   *  </strong>
+   * </p>
```

Comment:
The most  interesting part here is the semantics, that the actor unsubscribes to all subscriptions created for it, not entirely sure the performance caveeat is needed.